### PR TITLE
feat: Adding documentation for OpenSearch installations

### PR DESCRIPTION
**Pull request by grlimacan**
## Description

Adding documentation for OpenSearch installations. 

C7 epic: https://github.com/camunda/product-hub/issues/2233
C8 epic: https://github.com/camunda/product-hub/issues/1908

Relates to https://github.com/camunda/camunda/issues/23087

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by grlimacan** [comment]
Hey @camunda/tech-writers , I am very sorry for sending this into review so late, however due to release issues it was physically impossible to get this done any sooner. I hope one of you can still find the time to review this. @matthewBearCamunda will be doing the technical review from the Optimize side. Thanks!

File: optimize/self-managed/optimize-deployment/configuration/security-instructions.md [hunk 1]
```diff
@@ -59,11 +59,11 @@ Authorization controls what data a user can access and change in Optimize once a
 
 </TabItem>
 
-<TabItem value='elasticsearch'>
+<TabItem value='database'>
 
-Optimize stores its data in Elasticsearch, which is a search engine that acts as a document-based datastore. To protect access to this data, Elasticsearch should be configured carefully as well. Refer to the official [Secure the Elastic Stack](https://www.elastic.co/guide/en/elasticsearch/reference/master/secure-cluster.html#secure-cluster) documentation of Elasticsearch.
+Optimize stores its data in Elasticsearch or OpenSearch, which is a search engine that acts as a document-based datastore. To protect access to this data, the database should be configured carefully as well. Refer to the official security guidelines for [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/secure-cluster.html#secure-cluster) or [OpenSearch](https://opensearch.org/docs/latest/getting-started/security).
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/configuration/security-instructions.md:64]: `Optimize stores its data in Elasticsearch or OpenSearch, which is a search engine that acts as a document-based datastore`

The grammar on the latter half of the sentence needs changed

`Optimize stores its data in Elasticsearch or OpenSearch, which are search engines that act as a document-based datastore`

File: optimize/self-managed/optimize-deployment/configuration/shared-elasticsearch-cluster.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Changing the value of `*.settings.index.prefix` after an instance was already ru
 :::
 
 :::note
-OpenSearch support is currently only available for `ccsm` mode. Moreover, OpenSearch support in Optimize is limited to data import and the raw data report. The remaining functionality will be delivered with upcoming patches.
+Not all Optimize features are supported when using OpenSearch as a database. For a full list of the features that are now supported, please refer to the [Camunda 7](https://github.com/camunda/issues/issues/705) and [Camunda 8](https://github.com/camunda/issues/issues/635) OpenSearch features.
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/configuration/shared-elasticsearch-cluster.md:22]: Change to 

`Not all Optimize features are supported when using OpenSearch as a database. For a full list of the features that are currently supported, please refer to the`

File: optimize/self-managed/optimize-deployment/configuration/system-configuration.md [hunk 3]
```diff
@@ -191,7 +191,7 @@ Define a secured connection to be able to communicate with a secured Elasticsear
 These settings are only relevant when operating Optimize with OpenSearch.
 
 :::note
-OpenSearch support is currently only available for `ccsm` mode. Moreover, OpenSearch support in Optimize is limited to data import and the raw data report. The remaining functionality will be delivered with upcoming patches.
+Not all Optimize features are supported when using OpenSearch as a database. For a full list of the features that are now supported, please refer to the [Camunda 7](https://github.com/camunda/issues/issues/705) and [Camunda 8](https://github.com/camunda/issues/issues/635) OpenSearch features.
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/configuration/system-configuration.md:194]: Same as above, use `currently supported`

File: optimize/self-managed/optimize-deployment/install-and-start.md [hunk 4]
```diff
@@ -54,12 +54,12 @@ or `elasticsearch-startup.bat` on Windows:
 .\elasticsearch-startup.bat
 ```
 
-#### Production distribution without Elasticsearch
+#### Production distribution without database
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/install-and-start.md:57]: `#### Production distribution without a database`

File: optimize/self-managed/optimize-deployment/install-and-start.md [hunk 5]
```diff
@@ -54,12 +54,12 @@ or `elasticsearch-startup.bat` on Windows:
...
 
 This distribution is intended to be used in production. To install it, take the following steps:
 
-1. [Download](https://docs.camunda.org/enterprise/download/#camunda-optimize) the production archive, which contains all the required files to startup Camunda Optimize without Elasticsearch.
-2. [Configure the Elasticsearch connection](./configuration/getting-started.md#elasticsearch-configuration) to connect to your pre-installed Elasticsearch instance and [configure the Camunda 7 connection](./configuration/getting-started.md#camunda-platform-7-configuration) to connect Optimize to your running engine.
+1. [Download](https://docs.camunda.org/enterprise/download/#camunda-optimize) the production archive, which contains all the required files to startup Camunda Optimize without database.
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/install-and-start.md:61]: `Camunda Optimize without a database.`

File: optimize/self-managed/optimize-deployment/install-and-start.md [hunk 6]
```diff
@@ -85,10 +85,14 @@ Password: ******
...
+After that, [configure the database connection](./configuration/getting-started.md#elasticsearchopensearch-configuration) to connect to your pre-installed Elasticsearch/OpenSearch instance and [configure the Camunda connection](./configuration/getting-started.md#camunda-platform-7-configuration) to connect Optimize to your running engine. For very simple use cases with only one Camunda Engine and one database node, you can use environment variables instead of mounting configuration files into the Docker container:
 
 #### Getting started with the Optimize Docker image
 
+:::note
+Not all Optimize features are supported when using OpenSearch as a database. For a full list of the features that are now supported, please refer to the [Camunda 7](https://github.com/camunda/issues/issues/705) and [Camunda 8](https://github.com/camunda/issues/issues/635) OpenSearch features.
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/install-and-start.md:93]: `that are currently supported`

File: optimize/self-managed/optimize-deployment/install-and-start.md [hunk 7]
```diff
@@ -110,20 +122,42 @@ docker run -d --name optimize -p 8090:8090 -p 8091:8091 \
            registry.camunda.cloud/optimize-ee/optimize:{{< currentVersionAlias >}}
 ```
 
+Alternatively for OpenSearch:
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/install-and-start.md:125]: `Alternatively, for OpenSearch:`

add a comma

File: optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.13-to-3.14.md [hunk 8]
```diff
@@ -21,7 +21,7 @@ Here you will find information about:
 
 ## Limitations
 
-Not all features of Optimize are yet supported when using it with OpenSearch as a database. For a full list of the features that are now supported, please refer to the [Camunda 7 OpenSearch features](https://github.com/camunda/issues/issues/705).
+Not all Optimize features are supported when using OpenSearch as a database. For a full list of the features that are now supported, please refer to the [Camunda 7 OpenSearch features](https://github.com/camunda/issues/issues/705).
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.13-to-3.14.md:24]: `that are currently supported,`

File: optimize/self-managed/optimize-deployment/migration-update/camunda-8/3.13_8.5-to-8.6.md [hunk 9]
```diff
@@ -19,7 +19,7 @@ Here you will find information about:
 
 ## Limitations
 
-Not all features of Optimize are yet supported when using it with OpenSearch as a database. For a full list of the features that are now supported, please refer to the [Camunda 8 OpenSearch features](https://github.com/camunda/issues/issues/635).
+Not all Optimize features are supported when using OpenSearch as a database. For a full list of the features that are now supported, please refer to the [Camunda 8 OpenSearch features](https://github.com/camunda/issues/issues/635).
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/migration-update/camunda-8/3.13_8.5-to-8.6.md:22]: `that are currently supported,`

File: optimize/self-managed/optimize-deployment/install-and-start.md [hunk 10]
```diff
@@ -110,20 +122,42 @@ docker run -d --name optimize -p 8090:8090 -p 8091:8091 \
...
+
+```
+docker run -d --name optimize -p 8090:8090 -p 8091:8091 \
+           -e OPTIMIZE_CAMUNDABPM_REST_URL=http://yourCamBpm.org/engine-rest \
+           -e CAMUNDA_OPTIMIZE_DATABASE=opensearch \
+           -e CAMUNDA_OPTIMIZE_OPENSEARCH_HOST=yourOpenSearchHost \
+           -e CAMUNDA_OPTIMIZE_OPENSEARCH_HTTP_PORT=9200 \
```
Comment (matthewBearCamunda) [inline optimize/self-managed/optimize-deployment/install-and-start.md:132]: Change the port to `9205` as that is listed as the new default value

**Review by matthewBearCamunda — APPROVED** [review]
Pre-approving as the changes are all straightforward

Mostly minor grammatical changes

**Review by conceptualshark — COMMENTED** [review]
Echoing the above suggestions - happy to approve and get this merged once those addressed, just re-request review when ready. 👍

**Comment by grlimacan** [comment]
> Echoing the above suggestions - happy to approve and get this merged once those addressed, just re-request review when ready. 👍

Hey @conceptualshark, I have just implemented the suggestions, please re-review. Preview environment is not working for some technical reason that seems unrelated to my PR

**Review by conceptualshark — APPROVED** [review]
👍 LGTM!

**Pull request merged**